airflow/models/dagbag.py

```diff
@@ -316,7 +316,14 @@ def _load_modules_from_file(self, filepath, safe_mode):
         if mod_name in sys.modules:
             del sys.modules[mod_name]
 
-        timeout_msg = f"DagBag import timeout for {filepath} after {self.DAGBAG_IMPORT_TIMEOUT}s"
+        timeout_msg = (
+            f"DagBag import timeout for {filepath} after {self.DAGBAG_IMPORT_TIMEOUT}s.\n"
```

Comment:
let's use `get_docs_url` from `from airflow.utils.docs import get_docs_url`

e.g:

```
doc_link = get_docs_url("stable-rest-api-ref.html")
```

Reply:
Good point.  Let me fix it tomorrow, when I recover from ApacheCon Whiskey Birds of the Feather sesion ;)

Reply:
Recovered :)